.github/workflows/bench-turbopack-scheduled.yml

```diff
@@ -0,0 +1,137 @@
+name: Benchmark Turbopack
+
+on:
+  schedule:
+    # Run every week on Sunday 12:42
+    - cron: "42 12 * * 0"
+  pull_request:
+    branches: [main]
```

Comment:
File name is `-scheduled.yml` but also running on PR?

Reply:
Only for this condition: `if: github.event_name == 'push' || startsWith(github.head_ref, 'bench/')`

Reply:
but I guess it should be `github.event_name == 'pull_request'` instead

Reply:
fixed